capstone/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -30,16 +30,18 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
 
-    String loginUrl = userService.createLoginURL("/group.html");
+    String loginUrl = userService.createLoginURL("/");
     String logoutUrl = userService.createLogoutURL("/");
 
     String json;
     if (userService.isUserLoggedIn()) {
-      json = String.format("{\"email\": \"%s\", \"url\": \"%s\"}",
-                      userService.getCurrentUser().getEmail(), logoutUrl);
+      json = String.format(
+        "{\"email\": \"%s\", \"loginurl\": \"%s\", \"logouturl\": \"%s\", \"loggedIn\": %b}",
+        userService.getCurrentUser().getEmail(), loginUrl, logoutUrl, true);
     } else {
-      json = String.format("{\"email\": \"%s\", \"url\": \"%s\"}", 
-                          "null", loginUrl);
+      json = String.format(
+        "{\"email\": \"%s\", \"loginurl\": \"%s\", \"logouturl\": \"%s\", \"loggedIn\": %b}",
+        "null", loginUrl, logoutUrl, false);
```

Comment:
I recommend talking to Lucy about this since I know she did it for her portfolio, but instead of creating a string representation of the object, which can get super messy and can get hard to make sure the syntax is correct, can we use an object or some other data structure that contains the necessary information, convert that to JSON, and you send that over?

These may help:
https://github.com/lxcyqx/step/blob/master/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java#L90
https://github.com/lxcyqx/step/blob/master/portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java#L63

Reply:
Yes, that makes sense! Andrew and I mentioned this during our 1:1 too